### DISCUSSION TITLE
KNEC Courses

### DISCUSSION THREAD

**Discussion by DrWambua** [community/community#147376] [Programming Help]
Question

Who might have a JSON file containing information about Kenya National Examinations Council (KNEC) courses, including their titles, codes, and modules?

**Comment by Logophoman** [comment]
Usually I'd say we could scrape their website but I haven't even found a list of courses there, so probably easiest to contact them for their Programmes and then build a script that extracts the courses and build the JSON yourself. I'd write them an email or give them a call...

**Comment by mecodeatlas** [comment]
Thanks for posting in the GitHub Community, @DrWambua!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the Programming Help category. I've gone ahead and moved it for you. Good luck!
